src/components/Settings/Settings.js

```diff
@@ -30,18 +42,122 @@ export default class Settings extends React.Component {
             <option value="TestNet">TestNet</option>
             <option value="CozNet">CozNet</option>
             <option value="nOSLocal">nOS Local</option>
+
+            {
+              this.props.allNetworks.map((network) => {
+                return (
+                  <option
+                    key={network.neoscan}
```

Comment:
Should the key be `network.name` instead?  It seems like it could be possible for two networks to point to the same neoscan URL, which would make these non-unique.